contrib/mesos/pkg/scheduler/plugin.go

```diff
@@ -278,7 +278,11 @@ func (k *kubeScheduler) Schedule(pod *api.Pod, unused algorithm.NodeLister) (str
 			log.Infof("aborting Schedule, pod has been deleted %+v", pod)
 			return "", noSuchPodErr
 		}
-		return k.doSchedule(k.api.tasks().Register(k.api.createPodTask(ctx, pod)))
+		task, err := k.api.tasks().Register(k.api.createPodTask(ctx, pod))
```

Comment:
Also Register has this strange type with an error as second argument. Can we fix this as well here?